modules/fbx/readme.md

```diff
@@ -0,0 +1,196 @@
+# Open Source FBX Specification for the Importer
```

Comment:
I'd rename `readme.md` to `README.md`, which is more common.

Reply:
done :)